config/database.yml

```diff
@@ -1,30 +1,28 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  adapter: mysql2
+  encoding: utf8mb4
   timeout: 5000
+  database: <%= ENV.fetch('DATABASE_NAME') %>
+  host: <%= ENV.fetch('DATABASE_HOST', 'localhost') %>
+  port: <%= ENV.fetch('DATABASE_PORT', '3306') %>
+  username: <%= ENV.fetch('DATABASE_USERNAME') %>
+  password: <%= ENV['DATABASE_PASSWORD'] %>
```

Comment:
Hello reviewer! You may notice some things `#fetch` from ENV, and others do not. `#fetch` allows a default to be returned, as on line 7. If the default is absent, and the environment variable does not exist, `#fetch` will throw an error. For instance, if `$DATABASE_USERNAME` is missing, an error will be thrown at line 8. Finally, using hash access, like line 9, will return `nil` if the environment variable is not present.